examples/SMAC4HPO_rf.py

```diff
@@ -13,10 +22,22 @@
 from smac.scenario.scenario import Scenario
 from smac.facade.smac_hpo_facade import SMAC4HPO
 
-boston = load_boston()
+INSTANCE_PATH = 'data/instances/'
+
+
+def load_data_from_file(container_path: str, delimiter=','):
```

Comment:
I used the local data that is already a part of the examples. I only split it into multiple instances. It can serve as an example for passing multiple instances as budget to successive halving too.

Reply:
I see your point. However, it looks more complicated than before and thus, it is more unlikely that new users will really read these. Please try to make examples short and sweet.